src/brewHandler.h

```diff
@@ -314,9 +322,17 @@ void brew() {
         case kWaitBrew: // waiting time brew
             lastBrewTime = timeBrewed;
 
+            if (FEATURE_SCALE == 1) {
+                weightBrew = weight - weightPreBrew;
+            }
+
```

Comment:
Why does this have to be done here explicitly? Is it not handled by the general block above in line 250 already?